src/SelfService/Domain/Services/AuthorizationService.cs

```diff
@@ -247,7 +319,20 @@ public async Task<bool> CanRetryCreatingMessageContract(PortalUser portalUser, M
         }
         var kafkaTopic = await _kafkaTopicRepository.Get(messageContract.KafkaTopicId);
         bool isMember = await _membershipQuery.HasActiveMembership(portalUser.Id, kafkaTopic.CapabilityId);
-        bool isCloudEngineer = portalUser.Roles.Any(role => role == UserRole.CloudEngineer);
+        bool isCloudEngineer = false;
+
+        if (portalUser.Roles.Any(role => role == UserRole.CloudEngineer))
```

Comment:
We have this particular code in multiple parts; Should we extract it and thus simplify these sections? :)